iac/terraform/aws/web-client/local.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  web-client-domain = var.environment == var.PROD_ENVIRONMENT ? var.bloodconnect_domain : "${var.environment}.${var.bloodconnect_domain}"
```

Comment:
are you sure this is correct? Why is this var.PROD_ENVIRONMENT
why are we passing the same twice

Reply:
Correction done
Commit: https://github.com/CraftsmenLtd/BloodConnect/pull/119/commits/06ca6060feb5fc90993b09484a60a1cfa6d0748e